### PR TITLE
Dirt simple scaffolding

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: commands/new.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+  await execa('git', ['init'], { cwd: dest });
+
+  await execa(
+    'git',
+    [
+      'remote',
+      'add',
+      'origin',
+      `git@github.com:4catalyzer/${repoName(name)}.git`,
```
Comment (itajaja) [inline commands/new.js]: parametrize the org maybe?

File: templates/.gitignore [hunk 2]
```diff
@@ -0,0 +1,62 @@
+lib/
```
Comment (taion) [inline templates/.gitignore]: Could be fancy and pull down https://github.com/github/gitignore/blob/master/Node.gitignore, then add our own special `lib/`, `es/`, and `build/` (or dist? whatever we call it) ignores.

Okay sorry that's an outright dumb idea

File: commands/new.js (hunk 1 above)
Comment (jquense) [reply commands/new.js]: you think we'd use it for bfly or qsi? i already have qsi tooling, not sure i can write in terms of each other tho maybe

File: commands/new.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    eslint['eslint-plugin-jsx-a11y'] = '^5.1.1';
+  } else {
+    eslint['eslint-config-4catalyzer'] = '^0.3.3';
+  }
+
+  fs.writeJSON(
+    path.join(dest, 'package.json'),
+    {
+      name: a.name,
+      version: '1.0.0',
```
Comment (taion) [inline commands/new.js]: starting major version at 1? fancy

Comment (jquense) [reply commands/new.js]: Sticking with the npm default :P

**Pull request merged**